requirements.txt

```diff
@@ -1,5 +1,23 @@
+appnope==0.1.0
```

Comment:
good to update this but i'm wondering if we're actually using all of these things in the app -- a few that jump out that i dont see any mention of in the code are `enum34` (we're using python3.5 so this shouldn't be necessary i think), `prompt-toolkit`, `Pygments`, etc. Any chance you `pip freeze`'d this to generate the list after having manually installed some packages into your virtualenv? or maybe it was done off of system python without the virtualenv?